### DISCUSSION TITLE
Manually import Qualys scan output (or any non-json file)

### DISCUSSION THREAD

**Discussion by gelven4sec** [reconmap/reconmap#129] [Q&A]
Hi,

From my understanding, Reconmap support manually importing scanners output, a bit like Faraday does, assuming from the integration which is marked as configured :
![image](https://user-images.githubusercontent.com/37241392/207625438-f82f67f8-4883-438d-aac2-9a4dd6ad20a1.png)
(Am I wrong ?)

So what's the way to do it ? I tried through the web client in "System" -> "Import data". But it doesn't seems to accept any thing else than a json file while the qualys scan output is int XML format.

I don't see any other import option.

Thanks in advance for helping and may this thread help others.

**Comment by nixehooked** [comment]
Hey, you found the solution ?

**Reply by gelven4sec** [reply]
Hi,

Sadly no. I went with Faraday instead.

**Reply by nixehooked** [reply]
Merci de ta réponse j'ai cru comprendre que tu étais Français, j'ai réussis à import les données de mon .nessus vendredi, ou automatiquement j'ai pu avoir toute les vulnérabilités sur Reconmap. Depuis impossible de comprendre pourquoi ça ne fonctionne plus ...
Faraday offre la possibilité sur une version community de parser ces données et générer des docx ?
En tout cas merci de ta réponse

**Reply by gelven4sec** [reply]
En effet je suis français. 

Je ne pourrais pas t'aider pour ton problème avec Reconmap du coup.

Pour Faraday, la version communautaire permet de parser le resultat de Qualys (et surement Nessus aussi), mais pour générer des rapports je crois que c'est une fonctionnalité payante.

**Comment by nixehooked** [comment]
Merci pour tes retours !

Le jeu. 2 mars 2023, 14:16, Joakim Pettersen ***@***.***> a
écrit :
